core/src/main/java/org/commonjava/indy/core/ctl/IspnCacheController.java

```diff
@@ -37,44 +43,49 @@
     @Inject
     private CacheProducer cacheProducer;
 
+    @Inject
+    private IndyObjectMapper mapper;
+
     private EmbeddedCacheManager cacheManager;
 
+    private Map<String, String> string2keyMapper;
+
     @PostConstruct
     private void setUp()
     {
         cacheManager = cacheProducer.getCacheManager();
+        string2keyMapper = new HashMap<>();
+        string2keyMapper.put( "content-index", "org.commonjava.indy.content.index.ISPFieldStringKey2StringMapper" );
+        string2keyMapper.put( "default", "org.commonjava.indy.pkg.maven.content.StoreKey2StringMapper" );
     }
 
-    public void clean( String name ) throws IndyWorkflowException
+    // only work for some caches for debugging
+    public String export( String cacheName, String key ) throws Exception
```

Comment:
Maybe we should add limited caches' name here and ban the other out.

Reply:
I think we can allow failure for this tool api. (It is bad in case we want to see something but find it is banned). Also, when adding new caches, we have to maintain the ban list. That is I whant to avoid.